src/metrics/include/metrics/v2/event.hpp

```diff
@@ -17,7 +17,14 @@
 #define HEADER_960919A5_5455_4DD2_BC68_EFBAEB228BB0_INCLUDED
```

Comment:
Now that I've made these changes, I'm pretty unsatisfied with the architecture of this whole file. In particular, I think the Operations shouldn't interact with EventStream, which behaves more similarly to the ClientContext in the grpc qps benchmark drivers. Instead, I'd like to evolve GrpcClient towards being a complete interface to the system, owning the channels and streams. Right now it feels a little shoehorned in. In that case we'd be able to actually use the EventStreams (or ClientContext) as tags in the completion queue, so the main goal of client threads would be to reap the cq(s) and take actions based on the tag returned.

That said, I think reworking that would make sense as part of the change to finally deprecate CSV metrics and streamline the whole system through grpc.